ci_framework/roles/edpm_build_images/tasks/install.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: Ensure directories are present
+  ansible.builtin.file:
+    path: "{{ cifmw_edpm_build_images_basedir }}/{{ item }}"
+    state: directory
+  loop:
+    - tmp
+    - artifacts/edpm_images
+
+- name: Ensure edpm-image-builder repository exist
+  register: eib_stats
+  ansible.builtin.stat:
+    path: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/edpm-image-builder"
+
+- name: Copy edpm-image-builder repository
+  ansible.builtin.copy:
+    src: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/"
+    dest: "{{ cifmw_edpm_build_images_basedir }}/tmp/"
+  when: eib_stats.stat.exists | bool
+
+- name: Get edpm-image-builder repository
+  ansible.builtin.git:
+    accept_hostkey: true
+    dest: "{{ cifmw_edpm_build_images_basedir }}/tmp/edpm-image-builder"
+    repo: "{{ cifmw_edpm_build_images_src }}"
+  when: not eib_stats.stat.exists
+
+- name: Install required packages
+  become: true
+  ansible.builtin.package:
+    name: "{{ cifmw_build_host_packages }}"
+    state: latest
+
+- name: Initialize python venv and install requirements
+  ansible.builtin.pip:
```

Comment:
We discussed to not use virtual env and use rpms.

Reply:
fyi.. steps i tried to make images without venv


* copy correct repos/ run reposetup role (I tried with zed repos but same steps would apply)

dnf install diskimage-builder
dnf install openstack-ironic-python-agent-builder

git clone https://github.com/openstack-k8s-operators/edpm-image-builder.git
cd edpm-image-builder/

export ELEMENTS_PATH="/home/zuul/edpm-image-builder/dib/:/usr/share/ironic-python-agent-builder/dib/"
diskimage-builder ./images/ironic-python-agent-centos-9-stream.yaml
diskimage-builder ./images/edpm-hardened-uefi-centos-9-stream.yaml